src/xtopcom/xrpc/xcluster/xcluster_query_manager.cpp

```diff
@@ -78,28 +78,36 @@ void xcluster_query_manager::getTransaction(xjson_proc_t & json_proc) {
     xdbg("xcluster_query_manager::getTransaction account: %s, tx hash: %s, version: %s", account.c_str(), tx_hash_str.c_str(), version.c_str());
     uint256_t tx_hash = hex_to_uint256(tx_hash_str);
     std::string strHash((char*)tx_hash.data(), tx_hash.size());
-    xtransaction_cache_data_t cache_data;
-    if (m_txstore != nullptr && m_txstore->tx_cache_get(strHash, std::make_shared<xtransaction_cache_data_t>(cache_data))) {
-    // if (m_transaction_cache != nullptr && m_transaction_cache->tx_get(strHash, std::make_shared<>(cache_data)) == 1) {
+
+    std::shared_ptr<xtransaction_cache_data_t> cache_data_ptr = std::make_shared<xtransaction_cache_data_t>();
```

Comment:
After 1.2.8, auditors won't support query, this func will be deleted later on.
Should be added in xarc_query_manager.cpp too.